libs/pika/runtime_configuration/src/runtime_configuration.cpp

```diff
@@ -68,7 +67,7 @@ namespace pika::detail {
         std::string r;
 
 #if defined(PIKA_WINDOWS)
-        PIKA_UNUSED(argv0);
+        [[maybe_unused]] argv0;
```

Comment:
As you probably noticed already, `[[maybe_unused]]` can't be used on expressions unfortunately so this will need one of the options mentioned in one of the other comments.